discussion/src/main/java/org/openedx/discussion/presentation/topics/DiscussionTopicsViewModel.kt

```diff
@@ -35,12 +37,15 @@ class DiscussionTopicsViewModel(
     val isUpdating: LiveData<Boolean>
         get() = _isUpdating
 
+    val hasInternetConnection: Boolean
+        get() = networkConnection.isOnline()
+
     var courseName = ""
 
-    fun updateCourseTopics() {
+    fun updateCourseTopics(withSwipeRefresh: Boolean) {
```

Comment:
Instead of this, could we simply call the `getCourseTopics()` method of the viewModel? Or would that cause any issues?

Reply:
Done, removed unnecessary code